### DISCUSSION TITLE
Manual Game Time or Autosplitting on Browser/Desktop/OBS versions?

### DISCUSSION THREAD

**Discussion by jB0ne90** [LiveSplit/LiveSplitOne#679] [Help]
Thought I'd ask this in here as I guess it's not really an 'issue' as such (other than to me). 😅

Is it currently possible to do Manual Game Time or Autosplitting in either the Browser, Desktop or OBS version? If so how? I'm on Mac (M1).

Any help much appreciated.

Cheers!

**Comment by CartoonFan** [comment]
I don't have an answer, but I'm curious as well. I'm on Arch Linux.

**Comment by KebabLord** [comment]
for manual splitting i'm using this script which sends the keypresses to the livesplit-one
https://github.com/thearst3rd/livesplitone-global-hotkeys

and for autosplitting, i modified this python script to send key signals when a level is completed (basically creating autosplitters using cheatengine / gameconqueror )
